object-browser-schema.js

```diff
@@ -0,0 +1,199 @@
+export default {
+  additionalProperties: false,
+  definitions: {
+    actionItemDisabled: {
```

Comment:
Why limit ourselves? We could set it up so you can have either the object or some shortcut string.